_analyzers/tokenizers/thai.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+title: Thai
+parent: Tokenizers
+nav_order: 140
+---
+
+# Thai tokenizer
+
+The `thai` tokenizer is designed for tokenizing Thai language text. Because words in Thai language are not separated by spaces, the tokenizer must identify word boundaries based on language-specific rules.
```

Comment:
```suggestion
The `thai` tokenizer tokenizes Thai language text. Because words in Thai language are not separated by spaces, the tokenizer must identify word boundaries based on language-specific rules.
```